src/SamplesApp/SamplesApp.Droid/SamplesApp.Droid.csproj

```diff
@@ -16,14 +16,12 @@
     <AndroidResgenFile>Resources\Resource.Designer.cs</AndroidResgenFile>
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <TargetFrameworkVersion>v12.0</TargetFrameworkVersion>
-    <AndroidUseLatestPlatformSdk>true</AndroidUseLatestPlatformSdk>
```

Comment:
You'll need to keep this particular entry, otherwise the build will fail on macOS.